src/components/borrowers/Quotes.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { StaticImage } from 'gatsby-plugin-image';
+import { QuoteA55 } from './quotes/QuoteA55';
+import { QuoteTecredi } from './quotes/QuoteTecredi';
+
+export const Quotes = () => {
+  const [quote, setQuote] = React.useState('a55');
+
+  const handleClickQuote = (quote: string) => {
+    console.log('cliecked')
+    if (quote === 'a55') {
```

Comment:
why the if? just do setQuote(quote), or even better, do it in the HTML instead of here?

Reply:
done